File: UVA/10209_Is_This_Integration/solution.py
import math

while True:
    try:
        radius = list(map(float, input().split()))
    except:
        break

    for i in range(len(radius)):
        striped_region = radius[i] * radius[i] * (math.pi / 3 - math.sqrt(3) + 1)
        dotted_region = 2 * radius[i] * radius[i] * (math.pi / 6 + math.sqrt(3) - 2)
        rest_region = 4 * radius[i] * radius[i] * (1 - math.pi / 6 - math.sqrt(3) / 4)
        
        print(f"{striped_region:.3f} {dotted_region:.3f} {rest_region:.3f}")

File: UVA/10209_Is_This_Integration/solution.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
    double radius;
    while (cin >> radius) {
        double striped_region = radius * radius * (M_PI / 3 - sqrt(3) + 1);
        double dotted_region = 2 * radius * radius * (M_PI / 6 + sqrt(3) - 2);
        double rest_region = 4 * radius * radius * (1 - M_PI / 6 - sqrt(3) / 4);

        cout << fixed << setprecision(3);
        cout << striped_region << " " << dotted_region << " " << rest_region << endl;
    }

    return 0;
}